### PR TITLE
Updated plugin to support new postcss@8 api, also fixed a bug with ps…

### PR DESCRIPTION
**Pull request by 4eb0da**
…eudo-classes iteration

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,7 +35,7 @@ var removeClasses = exports.removeClasses = function removeClasses(matchesClassN
         node.nodes = (0, _fp.reject)(parseNode, node.nodes);
         return false;
       case 'selector':
-        var didRemoveNodes = (0, _fp.some)(parseNode, node.nodes);
+        var didRemoveNodes = node.nodes.some(parseNode);
```
Comment (4eb0da) [inline index.js:38]: If collection is being modified inside iteration function, `some` from `lodash` will fail
Built-in `some` will work properly

File: test/index.js [hunk 2]
```diff
@@ -63,6 +63,7 @@ test('it should handle :has', (t) => {
 test('it should handle :not', (t) => {
   t.is(removeClasses(classNameMatches('a'), 'p:not(.a)'), 'p');
   t.is(removeClasses(classNameMatches('a'), 'p:not(p.a)'), 'p');
+  t.is(removeClasses(classNameMatches('a'), 'p:not(.a) .b'), 'p .b');
```
Comment (4eb0da) [inline test/index.js:66]: New test covers such case

**Comment by jacobp100** [comment]
Thanks! Sorry the stuff in this repo was a bit dated. I'll try and publish a new version tonight (will be semver major)

**Comment by 4eb0da** [comment]
Wow, i didn't expect any reaction so soon. Thanks a lot!

**Comment by jacobp100** [comment]
I aim to please! Published version 2

**Comment by 4eb0da** [comment]
Nice!

**Pull request merged**